$OpenBSD: patch-modules_demux_avi_avi_c,v 1.1 2011/05/08 16:20:16 dcoppa Exp $

Don't free uninitialized ES format descriptor for DV muxed in AVI.

--- modules/demux/avi/avi.c.orig	Thu May  5 20:34:10 2011
+++ modules/demux/avi/avi.c	Thu May  5 20:34:42 2011
@@ -643,7 +643,10 @@ static int Open( vlc_object_t * p_this )
         if( tk->p_out_muxed == NULL )
             tk->p_es = es_out_Add( p_demux->out, &fmt );
         TAB_APPEND( p_sys->i_track, p_sys->track, tk );
-        es_format_Clean( &fmt );
+        if(!p_sys->b_muxed )
+        {
+            es_format_Clean( &fmt );
+        }
     }
 
     if( p_sys->i_track <= 0 )
